mysql-test/t/ssl_san.test

```diff
@@ -16,11 +16,11 @@ GRANT ALL ON test.* TO 'mysqltest_1'@'127.0.0.0/255.0.0.0';
 # to another server ('CN' == 'percona-server'), it has 'localhost' and
 # 127.0.0.1 in the Subject Alternative Names (SAN) list.
 
---exec $MYSQL -h localhost --ssl-mode=VERIFY_CA --protocol=tcp --ssl-ca=$MYSQL_TEST_DIR/std_data/percona-cacert.pem -e "SELECT 'localhost' AS answer"
---exec $MYSQL -h localhost --ssl-mode=VERIFY_IDENTITY --protocol=tcp --ssl-ca=$MYSQL_TEST_DIR/std_data/percona-cacert.pem -e "SELECT 'localhost (--ssl-mode=VERIFY_IDENTITY)' AS answer"
+--exec $MYSQL -u mysqltest_1 -h localhost --ssl-mode=VERIFY_CA --protocol=tcp --ssl-ca=$MYSQL_TEST_DIR/std_data/percona-cacert.pem -e "SELECT 'localhost' AS answer"
```

Comment:
is the user relevent here?

Reply:
Yes, as the testcase creates it earlier for this purpose.